ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/Cli.scala

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool
+
+import scopt.{OParser, Read}
+
+object Cli {
+  def config(args: Array[String]): Option[Config] =
+    OParser.parse(parser, args, defaultConfig)
+
+  private val parser = {
+    val builder = OParser.builder[Config]
+    import builder._
+    import Reads._
+    OParser.sequence(
+      programName("ledger-api-bench-tool"),
+      head("A tool for measuring transaction streaming performance of a ledger."),
+      opt[String]("hostname")
```

Comment:
Please make ledger-api-bench-tool cli compatible with ledger-api-test-tool which takes server address(es) in the form `<host>:<port>`. We need to support just one atm